src/lib/cost_model/cost_model_logical.cpp

```diff
@@ -0,0 +1,45 @@
+#include "cost_model_logical.hpp"
+
+#include "abstract_cost_feature_proxy.hpp"
+#include "operators/abstract_operator.hpp"
+
+namespace opossum {
+
+std::string CostModelLogical::name() const { return "CostModelLogical"; }
+
+Cost CostModelLogical::get_reference_operator_cost(const std::shared_ptr<AbstractOperator>& op) const {
+  return estimate_operator_cost(op);
```

Comment:
Because for the logical Cost Model, the "estimated Cost given the actual input cardinalities" and "actual Cost" are the same. It's because the CostModelLogical is such a dumb thing.